readme.md

```diff
@@ -27,5 +27,5 @@ page. Place the blocks so that they are horizontally centered on the page.
 
 **Result**
 
-[UPDATE DEMO LINK](https://<your_account>.github.io/<repo_name>/) <br>
-[UPDATE TEST REPORT LINK](https://<your_account>.github.io/<repo_name>/report/html_report/)
+[UPDATE DEMO LINK](https://gulldan1996.github.io/layout_style-it-up/) <br>
+[UPDATE TEST REPORT LINK](https://gulldan1996.github.io/layout_style-it-up/report/html_report/)
```

Comment:
Link doesn't work.
You may have forgotten some of the steps.

Reply:
does it work?

Reply:
yes